controllers/ibmpowervscluster_controller.go

```diff
@@ -201,8 +201,7 @@ func (r *IBMPowerVSClusterReconciler) reconcile(ctx context.Context, clusterScop
 	var errList []error
 	// receive return values from the channel and decide the requeue
 	for val := range ch {
-		//nolint:staticcheck
-		if val.Requeue {
+		if val.RequeueAfter != 0 {
```

Comment:
Will there be a scenario where value of RequeueAfter is less than 0 or should the validation be changed to > 0?